README.md

```diff
@@ -131,6 +131,15 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [PepsiCo](https://www.pepsicojobs.com/internshipsprograms/jobs/241240?lang=en-us&previousLocale=en-US)| Purchase, New York <br> Plano, Texas | Software Development & Engineering Intern (No Sponsorship) |
 | [U.S. Bank](https://careers.usbank.com/global/en/job/2023-0020491/2024-Software-Engineering-Summer-Intern-Early-Careers) | Richfield, MN; and Atlanta, GA | 2024 Software Engineering Summer Intern |
 | Jane Street | New York, NY | [Software Engineer Internship](https://www.janestreet.com/join-jane-street/position/6834728002/) <br/> [Quantitative Trader Internship](https://www.janestreet.com/join-jane-street/position/6850721002/) <br/> [Quantitative Researcher Internship](https://www.janestreet.com/join-jane-street/position/6838982002/) |
+| [SkyWorks](https://careers.skyworksinc.com/job/Austin-Embedded-FW-Summer-Intern-2024-TX-73301/1054957600) | Austin, TX | Embedded FW Summer Intern |
+| [Cole Engineering Services, Inc.](https://workforcenow.adp.com/mascsr/default/mdf/recruitment/recruitment.html?cid=5055dea5-b670-4a19-adb1-2aea04ba5207&ccId=19000101_000001&type=MP&lang=en_US&jobId=472432) | Orlando, FL | Intern (Software) (US Citizenship Required) |
+| [Discover](https://jobs.discover.com/jobs/r292162024-business-technology-internship-program-riverwoods/) | Riverwoods, IL | Business Technology Internship Program (No Sponsorship) |
+| [Tyler Technologies](https://jobs.jobvite.com/tylertech/job/ojyMlfwI) | Moraine, OH | Software Development Intern - Summer/Fall |
+| [Steris](https://careers.steris.com/job/RICHMOND-Software-Engineering-Intern-CA-94804/1054522400/) | Richmond, CA | Software Engineer Intern |
```

Comment:
I'm unsure as to whether this one counts enough as a "tech" internship for this repo. The listed job duties sound mostly relevant, but it also asks for coursework in physics, EE, and embedded design.

Reply:
I don't see a problem with a posting like this. Not every software engineering internship needs to be super generic -- some SW internships might be focused on a certain area that requires that coursework (e.g., embedded SW).